### ISSUE TITLE
Game Executable still requested even though the Action exists already

### ISSUE THREAD

**Issue by Litcube** [azuravian/playnite-LocalInstallButton#10]
If the Play action exists, it shouldn't ask for the executable again.

**Comment by azuravian** [comment]
Just so I understand, are you saying you have incidents where the Play action already exists (meaning the game is already installed), but the game is marked as uninstalled, giving you the option to use this plugin to install?

**Comment by Litcube** [comment]
Nope, I'm refering to when the game has a play action and is uninstalled.  Reinstalling the game, it asks for the play action again when one already exists.  I changed this myself in the code, just thought I'd let you know.

**Comment by azuravian** [comment]
Been thinking about this again.  I'm assuming your own fix worked for you.  My concern for a broad fix is that there are edge cases where a game might have 2 Play Actions.  Thinking about adding a check for if user wants to use their existing Play action for this game, or if they'd like to create a new Play Action.

**Comment by Koriebonx98** [comment]
could do on install 
add all exe's other than thise in "_Redist" as play actions.

**Comment by azuravian** [comment]
This plugin will be replaced soon with a new one that operates as a library plugin instead of general.  The new one will process uninstalls, automatically removing any play options that were created.

**Comment by azuravian** [comment]
Will be shutting down this repository.  My plugin is now a Library plugin and is at 
https://github.com/azuravian/playnite-LocalLibrary

Check it out
